app/controllers/price_guide_search_controller.rb

```diff
@@ -2,23 +2,27 @@
 # Price Guide Controller.
 class PriceGuideSearchController < ApplicationController
```

Comment:
@bgdeutsch Please fix the indentation here. It was 2 spaces. There should be no changes to this file.